rfcs/text/0042/risk.yml

```diff
@@ -0,0 +1,168 @@
+---
+- name: risk
+  title: Risk information
+  group: 2
+  short: Fields for describing risk score and level.
+  beta: >
+    These fields are in beta and are subject to change.
+  description: >
+    Fields for describing risk score and risk level of entities such as hosts and users.
+    These fields are not allowed to be nested under `event.*`. Please continue to use
+    `event.risk_score` and `event.risk_score_norm` for event risk.
+  reusable:
+    top_level: false
+    expected:
+      - host
+      - user
+  type: group
+  fields:
+    - name: identifier_field
```

Comment:
ECS steers away from abbreviations, but the schema already uses [`id`](https://www.elastic.co/guide/en/ecs/current/ecs-host.html#field-host-id) if you wanted to shorten to `id_field` and `id_value`.

Reply:
This is an interesting one; while `id` refers to the elasticsearch identifier `_id`, `identifier_*` here have an [entirely different meaning](https://github.com/elastic/ecs/pull/2236/files#diff-40e16fa32db2ffbdeabf56eb2ffeac75dfa12410e662c3328c7c859b426805d2R34).

That's not to say that the unabbreviated name helps at all with that distinction 😅 . @SourinPaul unless you disagree I'm going to change these to `id_field` and `id_value`; I'm into the whole brevity thing.